ninja_jwt/schema.py

```diff
@@ -19,6 +19,9 @@
 user_name_field = get_user_model().USERNAME_FIELD  # type: ignore
 
 
+AuthUserSchema = create_schema(get_user_model(), fields=[user_name_field])

```

Comment:
@demiurg please can you switch to `ModelSchema` and remove the `ninja.orm` dependency

```python
class AuthUserSchema(ModelSchema):
      class Config:
        model = get_user_model()
        include = [user_name_field]
```
That will do the work

Thank for fixing this.

Reply:
Good idea!